analyses/snv-callers/run_caller_consensus_analysis-pbta.sh

```diff
@@ -49,31 +49,55 @@ python3 analyses/snv-callers/scripts/01-setup_db.py \
   --db-file $dbfile \
   --consensus-file $consensus_file
 
-###################### Create intersection BED files ###########################
-# Make All mutations BED file
+############# Create intersection BED files for TMB calculations ###############
+# Make All mutations BED files
 bedtools intersect \
   -a data/WGS.hg38.strelka2.unpadded.bed \
-  -b data/WGS.hg38.mutect2.vardict.unpadded.bed > $all_mut_wgs_bed
+  -b data/WGS.hg38.mutect2.vardict.unpadded.bed \
+  > $all_mut_wgs_bed
 
+#################### Make coding regions file 
 # Convert GTF to BED file for use in bedtools
 # Here we are only extracting lines with as a CDS i.e. are coded in protein
 gunzip -c data/gencode.v27.primary_assembly.annotation.gtf.gz \
   | awk '$3 ~ /CDS/' \
   | convert2bed --do-not-sort --input=gtf - \
-  > $cds_file
+  > scratch/coding_sequences.bed
+
+# Sort so we can merge
+sort -k 1,1 -k 2,2n coding_sequences.bed | 
```

Comment:
The output and inputs here don't match (scratch/coding_sequences.bed vs. coding_sequences.bed). Might be worth just piping the whole way

```gunzip -c data/gencode.v27.primary_assembly.annotation.gtf.gz \
  | awk '$3 ~ /CDS/' \
  | convert2bed --do-not-sort --input=gtf - \
  | sort -k 1,1 -k 2,2n \
  | bedtools merge -i stdin  \
  > $cds_file
```

Reply:
This did work, btw.